### PR TITLE
Move languageswitcher

### PR DESCRIPTION
**Pull request by mscharfstein**
Works on #13

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -18,7 +19,7 @@ function App() {
      * Retrieves catalog data from API and stores in Redux
      */
     async function fetchData() {
-      const resp = await fetch('/api/explore/v1/catalog');
+      const resp = await fetch("/api/explore/v1/catalog");
```
Comment (marcja) [inline client/src/App.js:22]: You got bit by the linter...

File: client/src/App.js [hunk 2]
```diff
@@ -31,7 +32,12 @@ function App() {
     fetchData();
   });
 
-  return <Form />;
+  return (
+    <div>
```
Comment (marcja) [inline client/src/App.js:36]: Nit: change `<div>` to `<section>`

File: client/src/components/QuestionForm.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ * @return {string} The top-level form with questions posed to users to
+ * determine which services they may be eligible for
+ */
+function QuestionForm() {
+  const form = useSelector(selectForm);
+  const sections = useSelector(selectSections);
+  const questions = useSelector(selectQuestions);
+  const { t } = useTranslation();
```
Comment (marcja) [inline client/src/components/QuestionForm.jsx:19]: Linter bites...

File: client/src/i18n.js [hunk 4]
```diff
@@ -14,15 +14,15 @@ i18next
       ns: ['translation', 'catalog'],
       defaultNS: 'translation',
       backend: {
-        backends: [
-          HttpBackend,
-          HttpBackend,
+        backends: [HttpBackend, HttpBackend],
```
Comment (marcja) [inline client/src/i18n.js:17]: revert incorrect linter changes

File: client/src/App.js (hunk 1 above)
Comment (mscharfstein) [reply client/src/App.js:22]: ugh. done

**Pull request merged**